.github/workflows/fake-bpy-ci.yml

```diff
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
```

Comment:
Please delete these commented lines for now.
After I checked on `ci_testing` branch, you can add a trigger for master and Pull Request :)

Reply:
removed